docs/workflow/requirements/linux-requirements.md

```diff
@@ -1,111 +1,117 @@
 
-These instructions are written assuming the current Ubuntu LTS.
+### Debian/Ubuntu
 
-Install the following packages for the toolchain:
+These instructions are written assuming the current *Ubuntu LTS*.
+
+The packages you need to install are shown in the following list:
 
-* CMake 3.20 or newer
-* llvm
-* lld
-* clang (for WASM 16 or newer)
-* build-essential
-* python-is-python3
-* curl
-* git
-* lldb
-* libicu-dev
-* liblttng-ust-dev
-* libssl-dev
-* libkrb5-dev
-* zlib1g-dev
-* ninja-build (optional, enables building native code with ninja instead of make)
-
-**NOTE**: If you have an Ubuntu version older than 22.04 LTS, or Debian version older than 12, don't install `cmake` using `apt` directly. Follow the note written down below.
+- `CMake` (version 3.20 or newer)
+- `llvm`
+- `lld`
+- `Clang` (see the [Clang for WASM](#clang-for-wasm) section if you plan on doing work on *Web Assembly (Wasm)*)
+- `build-essential`
+- `python-is-python3`
+- `curl`
+- `git`
+- `lldb`
+- `libicu-dev`
+- `liblttng-ust-dev`
+- `libssl-dev`
+- `libkrb5-dev`
+- `ninja-build` (Optional. Enables building native code using `ninja` instead of `make`)
+
+**NOTE:** If you are running on *Ubuntu* older than version *22.04 LTS*, or *Debian* older than version 12, then don't install `cmake` using `apt` directly. Follow the instructions in the [CMake on Older Versions of Ubuntu and Debian section](#cmake-on-older-versions-of-ubuntu-and-debian) later down in this doc.
 
 ```bash
 sudo apt install -y cmake llvm lld clang build-essential \
   python-is-python3 curl git lldb libicu-dev liblttng-ust-dev \
-  libssl-dev libkrb5-dev zlib1g-dev ninja-build
+  libssl-dev libkrb5-dev ninja-build
 ```
 
-**NOTE**: As of now, Ubuntu's `apt` only has until CMake version 3.16.3 if you're using Ubuntu 20.04 LTS (less in older Ubuntu versions), and version 3.18.4 in Debian 11 (less in older Debian versions). This is lower than the required 3.20, which in turn makes it incompatible with the repo. For this case, we can use the `snap` package manager or the _Kitware APT feed_ to get a new enough version of CMake.
-**NOTE**: If you have Ubuntu 22.04 LTS and older and your `apt` does not have clang version 16, you can add `"deb http://apt.llvm.org/$(lsb_release -s -c)/ llvm-toolchain-$(lsb_release -s -c)-18 main"` repository to your `apt`. See how we do it for linux-based containers [here](./../../../.devcontainer/Dockerfile).
+#### CMake on Older Versions of Ubuntu and Debian
+
+As of now, Ubuntu's `apt` only has until *CMake* version 3.16.3 if you're using *Ubuntu 20.04 LTS* (less in older Ubuntu versions), and version 3.18.4 in *Debian 11* (less in older Debian versions). This is lower than the required 3.20, which in turn makes it incompatible with the runtime repo. To get around this, there are two options you can choose: Use the `snap` package manager, which has a more recent version of *CMake*, or directly use the *Kitware APT Feed*.
```

Comment:
What do you mean by "less in older Ubuntu/Debian versions?

Reply:
That the CMake version those Ubuntu versions provide is less modern. Like for example, Ubuntu 22.04 offers 3.22.1, Ubuntu 20.04 only offers up to 3.16.3, and so on.